drivers/hmis_csv_importer/app/models/hmis_csv_importer/hmis_csv_cleanup/force_prioritized_placement_status.rb

```diff
@@ -0,0 +1,46 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HmisCsvImporter::HmisCsvCleanup
+  class ForcePrioritizedPlacementStatus < Base
+    def cleanup!
+      assessment_batch = []
+
+      assessment_scope.update(PrioritizationStatus: 1)
```

Comment:
This should probably use `update_all`  I think newer rails can use `update` but generally that's for single record updates.
```suggestion
      assessment_scope.update_all(PrioritizationStatus: 1)
```